iothub/service/src/DigitalTwin/DigitalTwinClient.cs

```diff
@@ -98,6 +98,7 @@ public static DigitalTwinClient CreateFromConnectionString(string connectionStri
 
         /// <summary>
         /// Creates an instance of <see cref="DigitalTwinClient"/>.
+        /// For more information on configuring IoT hub with Azure Active Directory, <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-dev-guide-azure-ad-rbac"/>
```

Comment:
This feels like a statement that belongs in a `<remarks>` tag. Thoughts?

Reply:
Good point. I'll move it to remarks.